gen/sam/sam_sa15.py

```diff
@@ -15,13 +15,18 @@ class SA15Generator(AirDefenseGroupGenerator):
     price = 55
 
     def generate(self):
-        self.add_unit(
-            AirDefence.Tor_9A331,
-            "ADS",
-            self.position.x,
-            self.position.y,
-            self.heading,
+        num_launchers = 2
+        positions = self.get_circular_position(
+            num_launchers, launcher_distance=120, coverage=360
         )
+        for i, position in enumerate(positions):
+            self.add_unit(
+                AirDefence.Tor_9A331,
+                "ADS#" + str(i),
+                position[0],
+                position[1],
+                position[2],
+            )
         self.add_unit(
             Unarmed.UAZ_469,
             "EWR",
```

Comment:
Uh, wat? Not a part of your change so shouldn't be addressed in this PR, but wat?

Reply:
In the SAM generators are about 3 different approaches on how to generate the groups. This is where I first tried to streamline it before I gave up on it and focused just on launch numbers instead. Here the number was 1 before, so it was just placed, and I copy posted a approach used in different SAM sites, since the old approach was only feasible for 1 launcher.

Reply:
This just creates X units in a circular position. There are 2 launchers, so its not that usefull, but it is used in a lot of places where more launchers are placed.

Reply:
The "wat" is specifically "why is this UAZ an EWR?"